utils/ccloud_library.sh

```diff
@@ -892,6 +888,7 @@ function ccloud::create_ccloud_stack() {
   CLUSTER_CLOUD="${CLUSTER_CLOUD:-aws}"
   CLUSTER_REGION="${CLUSTER_REGION:-us-west-2}"
   CLUSTER=$(ccloud::maybe_create_and_use_cluster "$CLUSTER_NAME" $CLUSTER_CLOUD $CLUSTER_REGION)
+  (($? != 0)) && { echo "$CLUSTER"; exit 1; }
```

Comment:
IMO a slightly expanded error message can help a bash runner find the error location.   We know the logical ids, but might not be very clear for users.

Reply:
@rspurgeon if the status is not 0, the `CLUSTER _is_ the error message. Are you saying we should add even more to it?

Reply:
Yes, I'm not sure that's enough to help a user narrow in on the error location for debugging. Something like "Failed to create (or use) cluster: $CLUSTER", or similar

Reply:
This is the output if cluster creation fails:

```
[9:12:46] ~/git/examples/ccloud/ccloud-stack(DEVX-2251) ✗: ./ccloud_stack_create.sh
This example uses real Confluent Cloud resources.
To avoid unexpected charges, carefully evaluate the cost of resources before launching the script and ensure all resources are destroyed after you are done running it.
Do you still want to run this script? [y/n] y
Do you also want to create a Confluent Cloud ksqlDB app (hourly charges may apply)? [y/n] n

Creating Confluent Cloud stack for service account demo-app-29130, ID: 138906.
Error: unknown flag: --cloud
Usage:
  ccloud kafka cluster [command]

Available Commands:
  create      Create a Kafka cluster.
  delete      Delete a Kafka cluster.
  describe    Describe a Kafka cluster.
  list        List Kafka clusters.
  update      Update a Kafka cluster.
  use         Make the Kafka cluster active for use in other commands.

Global Flags:
  -h, --help            Show help for this command.
  -v, --verbose count   Increase verbosity (-v for warn, -vv for info, -vvv for debug, -vvvv for trace).

Use "ccloud kafka cluster [command] --help" for more information about a command.
```

Reply:
I see an issue with the CLI here: Error: unknown flag: --cloud.   I'm testing with calling the functions directly as well